source/kubernetes/manage-app/access-ci-cd/index.html.md

```diff
@@ -40,23 +40,25 @@ The deployment process is as follows.
 
 1. Argo CD updates the Kubernetes deployment with the new image tag. Kubernetes then co-ordinates the deployment by creating new application containers using the new image tag and cleans up the old containers.
 
-After the Sync operation, Argo CD triggers the [post-sync Argo Workflow](https://github.com/alphagov/govuk-helm-charts/blob/main/charts/argo-services/templates/workflows/post-sync/workflow.yaml) to run [smokey tests](https://github.com/alphagov/smokey) for the app.
+After the Sync operation, Argo CD triggers the [post-sync Argo Workflow](https://github.com/alphagov/govuk-helm-charts/blob/main/charts/argo-services/templates/workflows/post-sync/workflow.yaml) to run [end-to-end tests](https://github.com/alphagov/govuk-e2e-tests) for the app.
 
 ## Continuous deployment of a release of a GOV.UK app
 
 The continuous deployment (CD) process is an automatic deployment process. A release of a GOV.UK app triggers the CD process, for example when we merge a pull request.
 
-A new release is automatically generated by our continuous integration (CI) pipeline when a user or process adds a new successful commit to the main branch. CI builds the new image and pushes that new image to AWS Elastic Container Registry (ECR).
+A new release is automatically generated by our continuous integration (CI) pipeline when a user or process adds a new successful commit to the main branch. GitHub Actions creates a GitHub release with a version number.
 
-CI then updates the image tag value for integration in the [govuk-helm-charts repo](https://github.com/alphagov/govuk-helm-charts/tree/main/charts/app-config/image-tags/integration) for the new release.
+Then GitHub Actions builds the new image and pushes that new image to AWS Elastic Container Registry (ECR) and calls the [deploy-image Argo Workflow](https://github.com/alphagov/govuk-helm-charts/blob/main/charts/argo-services/templates/workflows/deploy-image/workflow.yaml).
```

Comment:
```suggestion
Then GitHub Actions builds the new image and pushes that new image to GitHub Container Registry (GHCR) and calls the [deploy-image Argo Workflow](https://github.com/alphagov/govuk-helm-charts/blob/main/charts/argo-services/templates/workflows/deploy-image/workflow.yaml).
```

We made a change so we push images to ghcr.io, and AWS ECR is a pull through cache.